### PR TITLE
Grammatically updated the docs/source/index.rst and torch/csrc/jit/doc…

### PR DESCRIPTION
**Pull request by sunami09**
…s/serialization.md files

Fixes #ISSUE_NUMBER

Grammatically updated the docs/source/index.rst and torch/csrc/jit/doc…

image

### CODE REVIEW

**Comment by sunami09** [comment]
Hey I'm a new bee in open source contribution community, this is my first contribution, can anyone help me with this issue. Any support is appreciated. 
image

File: docs/source/index.rst [hunk 1]
```diff
@@ -14,15 +14,15 @@ Features described in this documentation are classified by release status:
...
   *Stable:*  These features will be maintained long-term and there should generally
   be no major performance limitations or gaps in documentation.
-  We also expect to maintain backwards compatibility (although
+  We also expect to maintain backward compatibility (although
   breaking changes can happen and notice will be given one release ahead
   of time).
 
   *Beta:*  These features are tagged as Beta because the API may change based on
-  user feedback, because the performance needs to improve, or because
+  user feedback because the performance needs to improve, or because
```
Comment (sunami09) [inline docs/source/index.rst:22]: these are small grammatical mistakes.

**Comment by sunami09** [comment]
[](https://user-images.githubusercontent.com/66564001/152121627-c145b42c-7a0f-4894-9755-e15a7c97ae31.png)Hey I'm a new bee in open source contribution community, this is my first contribution, can anyone help me with this issue. Any support is appreciated.
image

**Comment by ezyang** [comment]
These don't seem to be substantive improvements, not accepting.

**Pull request closed without merging**